### PR TITLE
global: file count/size indexing

### PR DESCRIPTION
**Pull request by lnielsen**

### CODE REVIEW

File: zenodo/modules/deposit/indexer.py [hunk 1]
```diff
@@ -70,3 +70,8 @@ def indexer_receiver(sender, json=None, record=None, index=None,
     else:
         json['_updated'] = record.updated
     json['_created'] = record.created
+
+    # Compute filecount and total file size
+    files = json.get('_files', [])
+    json['filecount'] = len(files)
+    json['filesize'] = sum([f.get('size', 0) for f in files])
```
Comment (slint) [inline zenodo/modules/deposit/indexer.py]: `filesize` as a field name might give the impression of searching for the individual file size (Example query: `filesize:>1000000`), except of course if one is thinking of a record as a single file/archive. Alternatives: `record_size`, `total_filesize`, `dataset_size`, `archive_size`.

Comment (krzysztof) [inline zenodo/modules/deposit/indexer.py]: I second the name change, more alternatives: ``size``, ``filessize``

Comment (lnielsen) [inline zenodo/modules/deposit/indexer.py]: I agree that it is kind of misleading, the reasoning behind the name was to ensure that all the file related fields are predictable and simple.... ``filename, filetype, filesize, filecount``.

I'd prefer either ``size`` or ``bytesize``. WDYT?

Comment (slint) [inline zenodo/modules/deposit/indexer.py]: I vote for `size` out of these two. Another one: `datasize`

Comment (krzysztof) [inline zenodo/modules/deposit/indexer.py]: bytesize :-1:
size, datasize :+1:

**Pull request merged**